nansat/exporter.py

```diff
@@ -124,8 +124,16 @@ def export(self, filename='', bands=None, rm_metadata=None, add_geolocation=True
 
         self.logger.debug('Export - OK!')
 
-    def export2thredds(self, filename, bands={}, metadata=None, mask_name=None, no_mask_value=64, rm_metadata=None,
-                       time=None, created=None):
+    def export2thredds(self,
+        filename,
+        bands={},
```

Comment:
I do not really like the empty dictionary as an argument default. 
Maybe you could replace it with None, and check if bands is None in line 197

Reply:
Sorry to show up uninvited, but I saw this in Slack and it's indeed a bad idea to use mutables as default arguments:
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Absolutely agree 👍 
Done